### PR TITLE
Merge <ConfirmBid> and <ConfirmFirstTimeBid> #trivial

### PR DESCRIPTION
**Pull request by yuki24**
This is an attempt to consolidate the logic in the `<ConfirmBid>` and `<ConfirmFirstTimeBid>` components. Let me know what you think!

#skip_new_tests

### CODE REVIEW

File: src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx [hunk 1]
```diff
@@ -1,312 +0,0 @@
...
-      title: "",
-      passProps: {
-        onSubmit: this.onBillingAddressAdded,
-        billingAddress: this.state.billingAddress,
-        navigator: this.props.navigator,
-      },
-    })
-  }
-
-  canPlaceBid() {
```
Comment (sweir27) [inline src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx]: #minor looks like this didn't make it up to the consolidated component, but the logic would change slightly anyways  so I'll make this part of a follow-up PR.

**Comment by sweir27** [comment]
Looks like this needs a rebase, but I think it's a 👍direction.

I noted one place where some logic I added in a previous PR got removed, but I think it's alright and I can follow up with something more comprehensive now that the components are merged.

File: src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx (hunk 1 above)
Comment (yuki24) [reply src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx]: I've added this back but feel free to update as you need.

**Comment by yuki24** [comment]
This should be good for review.

**Pull request merged**